components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -221,4 +371,105 @@ Should you have a need to revert back to IAM v1, it is quick and easy to do:
+{{% /info %}}
+
+In this example you will add a project rule to the project `project-devops`, created as part of the [project creation walkthrough]({{< relref "iam-v2-guide.md#creating-a-project" >}}).
+You will update projects to apply this new project rule, causing all matching ingested resources to be associated with the appropriate project.
+You will use the global project filter to filter your ingested data by `project-devops`.
+
+First, determine which ingested resources should belong to the project. In this example, we want to add the following ingested resources to `project-devops`:
+
+* Compliance reports with Chef Organization `devops`
+* Infrastructure nodes with Environment `dev`
+* actions on Chef Server `devops.pizza`
+
+You may want to verify that those filters work as expected using the search filter bars on the Event Feed, Client Runs, and
+Reports pages.
+
+Navigate to the project details page of `project-devops`, created in the previous section, either by heading directly to `https://{{< example_fqdn "automate" >}}/settings/projects/project-devops` or selecting the project name on the project list page.
+
+Select the `Create Rule` button to create a new project rule. Choose type `Node` then fill in the first condition's fields.
+Feel free to create fake ingest data that corresponds to the example json below, or come up with a condition that matches your existing data set.
+
+{{% warning %}}
+Compliance reports must be using audit cookbook 7.5+ in order to make use of all of the available project rule attributes. Older reports will only have environment and role available as attributes.
+{{% /warning %}}
+
+```json
+{
+  "id": "devops-rule",
+  "name": "devops rule",
+  "type": "NODE",
+  "project_id": "project-devops",
+  "conditions": [
+    {
+      "operator": "EQUALS",
+      "attribute": "CHEF_ORGANIZATION",
+      "values": [
+        "devops"
+      ]
+    }
+  ]
+}
+```
+
+![](/images/docs/create-project-rule.png)
+
+Save the rule. If you need to change the name or the conditions, select the project rule name on the project details page.
+
+You should see a message that says `Edits are pending: update projects to apply edits.` Select the `projects` link in the message to go back to the project list page.
+
+On the project list page, the button `Update Projects` should be enabled since a new rule has been added to a project. You should also see `Edits pending` in the Ingest Rules column of `project-devops` in the projects table. Select `Update Projects` and confirm the action in the modal that pops up.
+
+![](/images/docs/update-projects-button.png)
+
+Updating a project begins an operation that applies all pending rule edits and then moves ingested resources into the correct projects according to those latest changes. An ingested resource is moved into a project only if it matches any of the project's rule conditions.
```

Comment:
maybe _at least one_ instead of _any_?

Reply:
Agree to "at least one", but should say "at least one of the project's rules", not "at least one of the project's rule _conditions_".